src/v2/Apps/Consign/Routes/SubmissionFlow/ArtworkDetails/__tests__/ArtworkDetails.jest.tsx

```diff
@@ -43,6 +46,8 @@ const validFormWithSpaces = {
   depth: " 5 ",
   units: " cm ",
   provenance: "  provenance  ",
+  location: "  NY, USA  ",
```

Comment:
**question (non-blocking):** Why is this string surrounded by spaces?

Reply:
https://github.com/artsy/force/blob/9ef87a27ff8bfd67f0e27310a6573af3c23ad415/src/v2/Apps/Consign/Routes/SubmissionFlow/ArtworkDetails/__tests__/ArtworkDetails.jest.tsx#L272